CONTRIBUTING.md

```diff
@@ -18,7 +18,7 @@ As you may or may not know, the Operator-Controller project aims to deliver the
 The user experience captured in the OLM V1 PRD introduces many requirements that are best satisfied by a microservices architecture. The OLM V1 experience currently relies on two projects:
```

Comment:
```suggestion
The user experience captured in the OLM V1 PRD introduces many requirements that are best satisfied by a microservices architecture. The OLM V1 experience currently relies on two components:
```

Reply:
I think we need to address this improvement in a follow-up to not mix it with the mono repo updates.

But I am happy to address all of them in another PR. WDYT?

Reply:
I think that Per's comment is related to the mono repo change. Catalogd and the Operator Controller are no longer separate projects, but the two components that make up OLM v1.

In that case, I think it makes sense to keep this with the mono repo changes.